accounts/tests.py

```diff
@@ -1,3 +1,43 @@
-# from django.test import TestCase
+import pytest
+from django.contrib.auth.models import User # noqa
+from .models import UserAccount
+from . import signals # noqa
```

Comment:
Is `signals` also required? I can't see where it's used.

Reply:
the models file needs it, for some reason when I tried to run tests I had errors unless I imported it. Should I try removing the file and putting the signals in the models file?

Reply:
I see,
@ifireball do you have an idea what's cause this required import?

The `signals` file contains the following that probably in use by the test somehow but I'm not sure how:
```
from django.contrib.auth.models import User
from django.db.models.signals import post_save
from django.dispatch import receiver
from .models import UserAccount


@receiver(post_save, sender=User)
def create_user_account(sender, instance, created, **kwargs):
    if created:
        UserAccount.objects.create(user=instance)


@receiver(post_save, sender=User)
def save_user_account(sender, instance, **kwargs):
    instance.useraccount.save()

```

Reply:
One of the things I'm testing involves the signals, could it be that?
They're meant to create a connected UserAccount instance whenever a regular user is created, one thing I'm testing is whether that happens